internal/ingress/controller/controller.go

```diff
@@ -124,10 +124,8 @@ func (n *NGINXController) syncIngress(interface{}) error {
 	ings := n.store.ListIngresses(nil)
 	hosts, servers, pcfg := n.getConfiguration(ings)
 
-	if n.isLeader() {
-		klog.V(2).Infof("Updating ssl expiration metrics.")
-		n.metricCollector.SetSSLExpireTime(servers)
-	}
+	klog.V(2).Infof("Updating ssl expiration metrics.")
```

Comment:
should this also be moved to https://github.com/kubernetes/ingress-nginx/pull/4251/files#diff-71033b20e83348da90cc4ad44522573dR160?

Reply:
done